src/types/types.ts

```diff
@@ -396,9 +396,21 @@ export enum AssetTransfersOrder {
 export enum NftTokenType {
   ERC721 = 'ERC721',
   ERC1155 = 'ERC1155',
+  NO_SUPPORTED_NFT_STANDARD = 'NO_SUPPORTED_NFT_STANDARD',
+  NOT_A_CONTRACT = 'NOT_A_CONTRACT',
   UNKNOWN = 'UNKNOWN'
 }
 
+/**
+ * NftTokenTypes that are allowed as request inputs.
+ *
+ * @public
+ */
+export type InputNftTokenType =
```

Comment:
The `/types` directory is exported into the package namespace. We're only using `InputNftTokenType` as an internal type. Can you move this type into  `nft-api.ts` ?

Reply:
ouais